src/entities/course/ui/course-item/course-item.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames/bind';
+import dayjs from 'dayjs';
+import { CourseItemData } from '@/entities/course';
+import { Image } from '@/shared/ui/image';
+import { LinkCustom } from '@/shared/ui/link-custom';
+import { Subtitle } from '@/shared/ui/subtitle';
+
+import styles from './course-item.module.scss';
+
+const cx = classNames.bind(styles);
+
+export const CourseItem = ({
+  title,
+  language,
+  startDate,
+  detailsUrl,
+  buttonText,
+  iconSrc,
+}: CourseItemData) => {
+  const dateTime = dayjs(startDate).toISOString();
+
+  return (
+    <section className={cx('course-item')}>
+      <figure className={cx('icon-container')}>
+        <Image
+          className={cx('course-icon')}
+          src={iconSrc}
+          alt=""
+          aria-hidden="true"
+          data-testid="course-image"
+        />
+      </figure>
```

Comment:
I don't remember exactly where we landed on the `figure` topic. Maybe you recall something?

Reply:
i don't really remember but this code was written this way before so i almost ain't touched anything. let me know if you want to get rid of this